torch/_inductor/graph.py

```diff
@@ -366,6 +366,20 @@ def run_node(self, n: torch.fx.Node):
             else:
                 result = super().run_node(n)
 
+            # require the same stride order for dense outputs,
+            # so that user-land view() will not throw because inductor
+            # output different strides than eager
+            # long term the solution is to make view() always succeed
+            # with infallible strides.
```

Comment:
Just for understand (the PR is ok) - What is view with infallible strides?

Reply:
https://docs.google.com/document/d/1wWUd_hBoCwnMV2CnJNWxNFMRZYFyspV_iyI_1Nya7c0/edit#heading=h.lmmok2ikj8ad